### DISCUSSION TITLE
CPU utilization high

### DISCUSSION THREAD

**Discussion by braveboo** [lllyasviel/Fooocus#3103] [Q&A] [answered]
When I run Fooocus with A100-80G-12C (Intel CPU), its CPU utilization fluctuates normally, but when I run Fooocus with A100-80-12 (AMD CPU), its CPU utilization maxes out. What could be the reason for this?

**Comment by mashb1t** [comment, marked as answer]
see feedback in https://github.com/lllyasviel/Fooocus/issues/3104
